DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateProgress/_DelegateProgressSummary.cshtml

```diff
@@ -22,6 +22,14 @@
     </dd>
   </div>
 
+  <div class="nhsuk-summary-list__row details-list-with-button__row">
+    <dt class="nhsuk-summary-list__key">
+      Professional Registration Number
+    </dt>
+    <partial name="_SummaryFieldValue" model="@Model.ProfessionalRegistrationNumber" />
+    <dd class="nhsuk-summary-list__actions" />
+  </div>

```

Comment:
I feel like this belongs at a different point in the page. Mainly because I see "Supervisor" above this and think that the PRN could perhaps belong to the supervisor rather than the delegate.

I think it would be better at the top of this table, but I think it would be best if it was displayed along with the delegate name above - though that might be a bit too far.

Reply:
I agree, here's the new layout, which Kevin has approved:

![localhost_5001_TrackingSystem_Delegates_CourseDelegates_DelegateProgress_276699_returnPage=1 (6)](https://user-images.githubusercontent.com/70278044/157262200-6e5b8e8e-bb17-45f4-8d9c-2f2e2e026f40.png)